dss-collection-nft/lib/go/test/types.go

```diff
@@ -45,7 +46,8 @@ func parseCollectionGroupData(value cadence.Value) CollectionGroupData {
 		fields[0].ToGoValue().(uint64),
 		fields[1].ToGoValue().(string),
 		fields[2].ToGoValue().(string),
-		fields[3].ToGoValue().(bool),
+		fields[3].ToGoValue().(string),
+		fields[4].ToGoValue().(bool),
```

Comment:
Missing last field `endTime`. Could update createCollectionGroupTimebound spec to check this

Reply:
Will do that in a new PR. Good call.